Source/MillicastPublisher/Private/Media/AudioGameCapturer.cpp

```diff
@@ -249,6 +249,10 @@ TArray<Audio::FCaptureDeviceInfo>& AudioDeviceCapture::GetCaptureDevicesAvailabl
  */
  //-------------------------------------------------------------------------------------------------------------------------------------------------------------
 
+// When including Windows headers with Unreal Engine code the PreWindowsApi.h and PostWindowsApi.h headers should wrap the includes.
```

Comment:
As the comment says, do this when using windows headers in UE code otherwise you can end up with some very nasty and hard to track problems when things go wrong.